src/code.cloudfoundry.org/cf-tcp-router/config/config.go

```diff
@@ -81,6 +86,21 @@ func (c *Config) initConfigFromFile(path string) error {
 		c.DrainWaitDuration = DrainWaitDefault
 	}
 
+	if len(c.FrontendTLS) > 0 {
+
+		for _, v := range c.FrontendTLS {
+			if len(v.PrivateKey) == 0 || len(v.CertChain) == 0 {
+				return fmt.Errorf("Missing private key and/or cert chain")
+			}
+		}
+
+	} else {
+		c.FrontendTLS = append(c.FrontendTLS, FrontendTLSConfig{
+			CertChain:  "",
+			PrivateKey: "",
+		})
+	}
+
 	if c.BackendTLS.Enabled {
```

Comment:
I was expecting more tcp-router logic here (to do stuff with the new values), but 🤞 thats coming in a different commit/pr?

Can we get tests to ensure the config validation is working correctly when certs are provided, when frontend-tls is provided with no certs and when no frontend-tls is provided at all?